include/fastrtps/utils/StringMatching.h

```diff
@@ -47,9 +56,20 @@ class StringMatching
     static bool matchPattern(
             const char* pattern,
             const char* input);
+
+    static std::vector<std::string> split(
+            const std::string& str,
+            const std::string& delim);
+
+    static bool readUint32(
```

Comment:
I am missing the unit test of these new two public methods.

Also, it does not seem to be used in this PR. Do we need to include them?

Reply:
This function is used by three implementation of custom transports but they are in a external repository.

Reply:
Finally I decided to move this function to the external repository.

Reply:
Either `split` method is also moved to the external repository or unit test for this new public API should be added

Reply:
Done